recipe/meta.yaml

```diff
@@ -21,21 +21,22 @@ requirements:
     - python
     - pip
     - setuptools
+    - setuptools_scm >=3.4.3
     - wheel
   run:
+    - importlib-metadata <4.3  # flake 8 requires an older version of importlib-metadata
     - autopep8 >=1.6.0,<1.7.0
     - flake8 >=4.0.0,<4.1.0
     - jedi >=0.17.2,<0.19.0
-    - mccabe >=0.6.0,<0.7.0
-    - pluggy
+    - mccabe >=0.6.0,<0.7.0  # flake 8 requires a version less than 0.7.0
+    - pluggy >=1.0.0
     - python-lsp-jsonrpc >=1.0.0
     - pycodestyle >=2.8.0,<2.9.0
     - pydocstyle >=2.0.0
     - pyflakes >=2.4.0,<2.5.0
     - pylint >=2.5.0
-    - python >=3.6
+    - python
     - rope >=0.10.5
-    - setuptools >=39.0.0
```

Comment:
setuptools is still a [run dependency](https://github.com/python-lsp/python-lsp-server/blob/v1.5.0/pyproject.toml#L20).

Reply:
the `setuptools` dependency is included in the run section
https://github.com/AnacondaRecipes/python-lsp-server-feedstock/blob/d4289ed4722d5f28454d7529c5cd15128d7c3f70/recipe/meta.yaml#L40